draft-ecahc-moderation.md

```diff
@@ -270,11 +270,14 @@ create throw-away personas for such denial-of-service behavior.)
   processes and actions. Should this document define some boundaries
   for what the moderator team can do?
 
-The moderator team SHOULD create and maintain a public mailing list
-for the community to discuss both the general moderation criteria and
-individual moderation decisions. To not distract from the
-topic-oriented discussion on other IETF lists, such meta-discussions
-SHOULD be actively redirected to the moderation discussion list.
+The moderator team is responsible to the IETF Chair.  The IETF Chair
+MAY create or designate a forum to facilitate discussion about
+moderation, and refer interested parties to that forum.  All actions
+taken by the moderation team SHALL be reported to the IETF Chair,
+as well as to the person who is the target of the action.  All
+bans longer than fourteen (14) days SHALL be reported to the forum in
+which the person was banned, and in the case of a broad ban, to the
+community in a manner decided by the IESG.
```

Comment:
That 14-day text is much too prescriptive. Let's instead require that the moderation team must define gradual reporting of actions.

Reply:
I disagree.  This is a matter of accountability **of** the moderator team **to** the community, and they cannot specify their own terms in that regard.

Reply:
I agree with Eliot here. I think the day count is not too important. We can always try 14 days and adjust it in the inevitable update (it is difficult to imagine getting everything right the first time).

The reason my document gives the moderation team a lot of power is so that they don't have to use it most of the time.

Reply:
Rather than hard-coding 14 days, we probably need $REPORTING-INTERVAL so that it can be changed without a new RFC... But I agree with Eliot about accountability.

Reply:
I also agree about accountability. But I think it is enough to say that the mod team MUST adhere to reasonable and proportional reporting, rather than detailing the "how" of it.

Reply:
Going to have to give you an 🙄 eyeroll emoji on this one, sorry. 14 days is a long time, and doesn't really constrain the moderator team. We do have a pick a number to set the boundary.

6 seconds? 6 minutes? 6 days? All too short.

6 weeks? That is a really long time.

Reply:
My concern is the subject of a moderation action doing an appeal if it's been 15 days.

Reply:
@larseggert 

First, this isn't particularly about appeal.  Rather, it's about transparency.  14 days and everyone needs to know what happened.

But even if it were about appeals, we say that people should have the right to appeal moderator decisions.  So what's the problem?

Reply:
OK, if everyone wants to be prescriptive, I'm in the rough.